### DISCUSSION TITLE
Remove price table footer in the quote PDF version

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#16535] [How to] [answered]
hi,

Currently I am using the following template to generate the table for the qoute PDF. 

$(inventory : type=table columns=seq,comment1,qty,unit,total,tax , href=no , labels=SNo,Product Description,Qty,Unit,Product Price, Tax , width=10px,225px,50px,45px,105px,95px )$

When i use it in the Yetiforce 6.3, it wont generate the footer section in the price table. After updating to yetiforce 6.4, it generate separate row in the table footer section.
![image](https://user-images.githubusercontent.com/96602176/193187484-5e7bcf3c-761d-4923-9abb-f180f192f4ac.png)

 I wish to remove the table footer in the pdf version.

Could you please guide me to do the same. 

Thanks & Regards,
Rajasekaran.

**Comment by Ilamparithi-Tamizhanangu** [comment, marked as answer]
hi,

solved the issue by editing the textparser file.

Thanks.

**Comment by mariuszkrzaczkowski** [comment]
I added the showTableFoot parameter
https://github.com/YetiForceCompany/YetiForceCRM/commit/c3f7c05d85fe57b2b3b8b740042ca9f8bff4b0f4
